src/commands/fun/thisisfine.ts

```diff
@@ -0,0 +1,16 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = 'everything is fine...probably';
+	aliases = ['tif'];
+
+	run(msg: Message): Promise<Message> {
+		return msg.channel.send({ files: [{
+			attachment: `${__dirname}../../../../../assets/images/thisisfine.png`,
```

Comment:
Could we possibly alias assets? Like we do for `@lib` and `@root`

So then we could do... 
```suggestion
			attachment: `@assets/images/thisisfine.png`,
```

(and the same for tif lines [29](https://github.com/ud-cis-discord/SageV2/pull/208/files#diff-15a77f2575ef549c61f7fa5d38eeab07596ff5a944dd0e1abb2060429e6535b9R29) and [34](https://github.com/ud-cis-discord/SageV2/pull/208/files#diff-15a77f2575ef549c61f7fa5d38eeab07596ff5a944dd0e1abb2060429e6535b9R34))

Reply:
that one is Ben's fault

Reply:
i fixed it in s;f too

Reply:
Iirc the aliases won't work for file uploads because they result in a relative path and an absolute path is used. That's why `s;f` uses this same method.

Reply:
yeah josh

Reply:
Ouch, that's obnoxious

Reply:
I don't remember the whole problem but I know there was an issue with using the aliases in the Jenkins build because they were relative. I think it was something along the lines of having the aliases be relative to the root of the project but when used in Jenkins they were relative to the file or something like that. I just remember the fix was to use absolute paths and not worry about it.

Reply:
Fair enough